README.md

```diff
@@ -46,6 +46,8 @@ Lisk is a next generation crypto-currency and decentralized application platform
   sudo -u postgres psql -d lisk_main -c "alter user "$USER" with password 'password';"
   ```
 
+  **NOTE:** Database user requires privileges to `CREATE EXTENSION pgcrypto`.
```

Comment:
I found a migration bug. Even when executing the sentence added to the Readme, the node gets stuck at the point below:
```
[log] 2017-08-07 13:00:54 | error - error: ERROR:  function validatemembalances() does not exist
LINE 1: SELECT * FROM validateMemBalances()
                      ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
[log] 2017-08-07 13:00:54 | error - task(checkMemTables): SELECT * FROM validateMemBalances()
[log] 2017-08-07 13:00:54 | error - error: ERROR:  relation "rounds_exceptions" does not exist
LINE 1: SELECT * FROM rounds_exceptions;
                      ^
[log] 2017-08-07 13:00:54 | error - task(checkMemTables): SELECT * FROM rounds_exceptions;
[ERR] 2017-08-07 13:00:54 | BatchError: ERROR:  function validatemembalances() does not exist
LINE 1: SELECT * FROM validateMemBalances()
```